Components/Button/Button.html

```diff
@@ -0,0 +1,470 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./buttonstyles.css" />
+    <title>Button Component</title>
+  </head>
+  <body>
+    <h1>Buttons</h1>
+    <h2>Primary Buttons</h2>
+    <div class="buttons">
+      <button class="primary-btn">PRIMARY</button>
```

Comment:
create a default button with ```class= btn``` and use it on every other button component.

```suggestion
      <button class="btn btn-primary">PRIMARY</button>
```